csrankings-w.csv

```diff
@@ -148,6 +148,7 @@ Wei Zeng 0002,Florida International University,http://cis.fiu.edu/~wzeng,yKYRdd0
 Wei Zhang 0004,Peking University,http://sei.pku.edu.cn/~zhangw,NOSCHOLARPAGE
 Wei Zhang 0016,Fudan University,http://www.cs.fudan.edu.cn/?page_id=2019,NOSCHOLARPAGE
 Wei Zhang 0076,University of Central Florida,https://www.cs.ucf.edu/~wzhang,NOSCHOLARPAGE
+Wei Zhang 0173,Shandong University,https://faculty.sdu.edu.cn/cszhangwei,uJ65n8MAAAAJ

```

Comment:
Web link is dead.

Reply:
@emeryberger Dear Professor, I have fixed the link in the newest commit. Please check it again.